### PR TITLE
Remove phpunit whitelist exceptions

### PR DESCRIPTION
**Pull request by jordisala1991**
I am targeting this branch, because this is pedantic.

## Subject

I think coverage is not calculated correctly.

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -9,12 +9,11 @@
          processIsolation="false"
          stopOnFailure="false"
          syntaxCheck="false"
-         verbose="true"
```
Comment (greg0ire) [inline phpunit.xml.dist]: Why not?

Comment (jordisala1991) [inline phpunit.xml.dist]: Copied this file from SonataAdmin.

**Comment by greg0ire** [comment]
> Where is coveralls when you need it

Maybe you need to touch a php file

**Comment by jordisala1991** [comment]
lets try again

**Comment by jordisala1991** [comment]
I did change `BlockBundle` to `SonataBlockBundle` on `phpunit.xml.dist`

**Comment by greg0ire** [comment]
Thanks @jordisala1991

**Pull request merged**